test/e2e/framework/pv/pv.go

```diff
@@ -53,6 +53,10 @@ const (
 
 	// VolumeSelectorKey is the key for volume selector.
 	VolumeSelectorKey = "e2e-pv-pool"
+
+	// IsWindowsStorageClassAnnotation represents a StorageClass annotation that
+	// marks a class as the Windows compatible StorageClass for e2e tests.
+	IsWindowsStorageClassAnnotation = "e2e.storageclass.kubernetes.io/windows-compatible-class"
```

Comment:
is this an existing pattern? @msau42

Reply:
it is a new annotation we added for this test purpose.